word.js

```diff
@@ -0,0 +1,50 @@
+var Letter = require("./letter.js")
+
+
+function Word(guessWord) {
+  this.guessWord = guessWord;
+  this.guessWordLength = this.guessWord.length;
+  this.charArray = [];
+}
+
+var Word2 = function(randomWord) {
+  this.randomWord = randomWord;
+  this.randomSplitWord = randomWord.toLowerCase().split('');
+
+  this.guessWord = function() {
+    var answerArray = [];
+
+    for(var i = 0; i < this.randomWord.length; i++) {
+      if(this.randomWord[i]) {
+        answerArray.push(new Letter(this.randomWord[i].toLowerCase()));
+      } else {
+        answerArray.push(this.randomWord[i]);
+      }
+
+    }
+    return answerArray;
```

Comment:
this function can be refactored using a higher order function called map and a ternary operator, like so:
```javascript
return this.randomSplitWord.map(function(letter, index){
	return letter ? new Letter(letter.toLowerCase()) : letter
})